webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -2461,6 +2466,41 @@ var CURRENT_GROUP_DATA={};
                     _this.toggleMediaOptions(this);
 
                 });
+                $mck_text_box.keyup(function (){
+                    if (CURRENT_GROUP_DATA.CHAR_CHECK) {
+                        if (!(document.getElementById('mck-char-count'))) {
+                            warningText.innerHTML += '<span> | </span><span id="mck-char-count"></span>';
+                        }
+                        var remtxt;
+                        var str = ($mck_text_box.text().toString()).trim();
+                        var len = str.length;
+                        if (len > 199) {
+                            if (len > 256) {
+                                remtxt = "(" + MCK_LABELS['limit.remove'] + " " + (len - 256) + " " + MCK_LABELS['limit.characters'] + ")";
+                            } else {
+                                remtxt = "(" + (256 - len) + " " + MCK_LABELS['limit.characters'] + " " + MCK_LABELS['limit.remaining'] + ")";
+                            }
+                            document.getElementById('mck-char-count').innerHTML = remtxt;
+                            warningBox.classList.remove("n-vis");
+                            if (!(warningBox.classList.contains('mck-warning-slide-up'))) {
+                                warningBox.classList.add('mck-warning-slide-up');
+                                warningBox.classList.remove('mck-warning-slide-down');
+                            }
+                        } else {
+                          if (!(warningBox.classList.contains("n-vis"))) {
+                                if (!(warningBox.classList.contains('mck-warning-slide-down'))) {
```

Comment:
@hardikparashar Why adding two if statements consecutively if there are no else statement for both of them?